index.html

```diff
@@ -21,83 +22,283 @@
+              <div class="btn-2">
+                <button type="button">See Project</button>
+              </div>
+          </div>
+        </div>
+      </li>
+    </ul>
+       
       </section>
-    </main>
+            <!-- About myself page talks of my carrer as sof -->
+
+      <section id="about-page" class="about-page-1">
+        <div class="about-content">
+          <h1 class="about-text">About <br> Myself </h1> 
+          <p class="tonic-para">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me. 
+          </p>
+          <div class="social-media">
+            <h2 class="social-media-text">LET'S CONNECT</h2>
+            <div class="social-media-icons"></div>
+            <ul>
+              <li>
+                <a href="https://github.com/Kossi-stack">
+                  <img
+                    src="assets/github.jpg"
+                    class="icon"
+                    alt="git"/>
+                </a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/kossifioklou2406/">
+                  <img
+                    src="assets/Vector.jpg"
+                    class="icon"
+                    alt="linkedin"/>
+                </a>
+              </li>
+              <li>
+                <a href="https://google.com">
+                  <img
+                    src="assets/angellist.jpg"
+                    class="icon"
+                    alt="google"/>
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com">
+                  <img
+                    src="assets/twitter.jpg"
+                    class="icon"
+                    alt="twitter"/>
+                </a>
+              </li>
+              <li>
+                <a href="https://mediamarkt.es">
+                  <img
+                    src="assets/medium.jpg"
+                    class="icon"
+                    alt="mediamarket"/>
+                </a>
+              </li>
+            </ul>
+             <div class="btn-2">
+              <button type="button">Get my resume</button>
+            </div>
+           </div>
+          <div>
+            <h3>Languages</h3>
+            <div class="lang-list">
+              <ul>
+                <li>
+                  <img src="assets/Ellipse-1.png" class="img-list" alt="javascript">
+                </li>
+                <li>javaScript</li>
+                <li>
+                  <img src="assets/Ellipse-2.png" class="img-list" alt="html">
+                </li>
+                <li>html</li>
+                <li>
+                 <img src="assets/Ellipse-3.png" class="img-list" alt="css">
+                </li>
+                <li>css</li>
+            </ul>
+            </div>
```

Comment:
- [ ] Each language text appears to the right of its own image. Currently, the image is on top of each language text. Please refer to the screenshots below and make necessary updates.
yours
![image](https://user-images.githubusercontent.com/68167810/125174671-c684bd80-e1cf-11eb-8af3-5686753d13ae.png)

template's
![image](https://user-images.githubusercontent.com/68167810/125174613-4ceccf80-e1cf-11eb-8c96-2596ca102816.png)